test/SILOptimizer/di_property_wrappers.swift

```diff
@@ -219,8 +219,8 @@ func testRefStruct() {
     // CHECK-NEXT:   + payload alloc 42
     // CHECK-NEXT:   .. init value = 42
     // CHECK-NEXT:   + payload alloc 27
-    // CHECK-NEXT:   .. set value = 27
     // CHECK-NEXT:   - payload free 42
+    // CHECK-NEXT:   .. set value = 27
```

Comment:
The order of output here has changed - does it look correct? Do we need to suppress this for property wrappers?